main/lsp/LSPMessage.h

```diff
@@ -2,6 +2,7 @@
 #define RUBY_TYPER_LSP_LSPMESSAGE_H
 
 #include "common/Timer.h"
+#include "common/common.h"
```

Comment:
We redefine rapidjson's assert in terms of our ENFORCE, presumably to turn on rapidjson asserts in our debug builds, and we do it in bazel:

```
In file included from main/lsp/requests/sorbet_error.cc:1:
In file included from ./main/lsp/requests/sorbet_error.h:4:
In file included from ./main/lsp/LSPTask.h:4:
In file included from ./main/lsp/LSPMessage.h:6:
In file included from external/rapidjson/include/rapidjson/document.h:20:
In file included from external/rapidjson/include/rapidjson/reader.h:22:
In file included from external/rapidjson/include/rapidjson/encodedstream.h:19:
external/rapidjson/include/rapidjson/memorystream.h:49:22: error: use of undeclared identifier 'ENFORCE'
    Ch* PutBegin() { RAPIDJSON_ASSERT(false); return 0; }
                     ^
<command line>:10:29: note: expanded from here
#define RAPIDJSON_ASSERT(x) ENFORCE(x)
                            ^
```

https://github.com/sorbet/sorbet/blob/2f5da883e58ea1455e9054c3ef789d512d971e9b/third_party/rapidjson.BUILD#L4-L9

i just learned about this.

Reply:
Thus, it's not ok to include any rapidjson file if you have not also either included common.h or defined a suitable replacement for ENFORCE

Reply:
Yup, that was intentional. This way we can use our error handling mechanisms